README.md

```diff
@@ -19,9 +19,11 @@ docker-compose run metrics --help
 | prisma_api_url          | PRISMA_API_URL          | https://api.eu.prismacloud.io | Prisma API key                   |
 | prisma_api_key          | PRISMA_API_KEY          |                          | Prisma API key                        |
 | prisma_api_password     | PRISMA_API_PASSWORD     |                          | Prisma API password                   |
+| scc_org_id              | SCC_ORG_ID              |                          | Google SCC numeric organisation ID    |
+| scc_sources_regex       | SCC_SOURCES_REGEX       | `.`                      | Google SCC sources Display Name filter regexp |
 | graphite_host           | GRAPHITE_HOST           |                          | Graphite hostname                     |
 | graphite_port           | GRAPHITE_PORT           | `2003`                   | Graphite port                         |
-| graphite_prefix         | GRAPHITE_PREFIX         |                          | Graphite port                         |
+| graphite_prefix         | GRAPHITE_PREFIX         |                          | Graphite metrics prefix               |
```

Comment:
As a side observer, I'm a bit confused what's the difference between `graphite_prefix` and `compliance_prefix`. Not sure if it's expected that a user of this code should come with a pre-knowledge of this specific difference: if it's not the case, it's worth elaborating in the comment at the left side.